inaugurator/main.py

```diff
@@ -32,6 +32,7 @@
 parser.add_argument("--inauguratorDownload", nargs='+', default=[])
 parser.add_argument("--inauguratorIgnoreDirs", nargs='+', default=[])
 parser.add_argument("--inauguratorTargetDeviceCandidate", nargs='+')
+parser.add_argument("--inauguratorTargetDeviceLabel", default="BOOT")
```

Comment:
Add a 'help' argument to the three device detection methods that says that those 3 are mutually exclusive.